README.md

```diff
@@ -42,15 +42,14 @@ logging experience.
 
 The metalogger module is a very lightweight bridge/wrapper for a number of
 popular logging implementations:
-[npmlog](https://github.com/isaacs/npmlog), 
-[loggly](https://www.loggly.com/), 
-[sumologic](https://www.sumologic.com/), 
+[npmlog](https://github.com/isaacs/npmlog),
+[loggly](https://www.loggly.com/),
+[sumologic](https://www.sumologic.com/),
 [syslog/ain2](https://npmjs.org/package/ain2),
-[log](https://github.com/visionmedia/log.js), 
-[util](http://nodejs.org/api/util.html). A node.js module that 
-uses the metalogger library can choose which logging implementation to use at runtime.
+[log](https://github.com/visionmedia/log.js),
+[util](http://nodejs.org/api/util.html).
```

Comment:
This used to be a list, diff shows you removed "-" characters and it's not a list, anymore? If, true - please keep the list.

Also, let's add an item for "winston" or even "any other logging facility supported by \[winston\]\(link\)